edge-hub/watchdog/src/child.rs

```diff
@@ -74,11 +74,15 @@ impl ChildProcess {
 }
 
 pub fn run(
-    name: String,
-    program: String,
-    args: String,
+    name: impl Into<String>,
+    program: impl Into<String>,
+    args: impl Into<String>,
     should_shutdown: Arc<AtomicBool>,
 ) -> Result<JoinHandle<()>> {
+    let name = name.into();
+    let program = program.into();
```

Comment:
nit: for those that you used only once you can inline `xxx.into()` calls without hurting readability.

Reply:
I personally think this is more readable. I didn't like how we were putting the into() function call into other logic.

Reply:
You'll get used to it soon 🤣